app/javascript/legacy_react/src/components/common/LabeledFieldComponent.tsx

```diff
@@ -10,7 +10,7 @@ export interface LabeledFieldComponentProps
     labelText: string
     inError:boolean
     error?:string
-    inStickyError?:boolean
+    inStickyError?:boolean | any
```

Comment:
What the reasoning here for allowing `inStickyError` to be `any`? It seems like it should be a boolean.